CRM/Mailing/BAO/Mailing.php

```diff
@@ -128,8 +128,9 @@ public static function getRecipients($mailingID) {
     $recipientsGroup = $excludeSmartGroupIDs = $includeSmartGroupIDs = $priorMailingIDs = array();
     $dao = CRM_Utils_SQL_Select::from('civicrm_mailing_group')
              ->select('GROUP_CONCAT(entity_id SEPARATOR ",") as group_ids, group_type, entity_table')
-             ->where('mailing_id = #mailing_id AND entity_table IN ("civicrm_group", "civicrm_mailing")')
+             ->where('mailing_id = #mailing_id AND entity_table IN ("!groupTableName", "civicrm_mailing")')
```

Comment:
@monishdeb @seamuslee001  I have no mechanism for demonstrating the frequency/distribution in real-world scenarios, but the list of valid values for `civicrm_mailing_group.entity_table` allows any mix of  `civicrm_group`, `civicrm_group_en_US`, `civicrm_group_fr_FR`, `civicrm_mailing` -- e.g. one might get a mix by:

* Interleaving steps of (a) composing/reusing mailings and (b) toggling multilingual support.
* Composing/previewing/delivering messages with different users (and different default locales).

Shouldn't this really be a pattern-match?

```php
->where('mailing_id = #mailing_id AND entity_table RLIKE "^civicrm_(group.*|mailing)$"')
```

(_Aside_: The data model is evil.)

Reply:
I guess that could work. I also wonder do we need the AND entity_table IN etc if we can only have 2 types of values groups and mailings and we are including both here.

Reply:
We should probably figure out how we can add appropriate testing for these things since otherwise we are always gonna hit more

Reply:
Ahh right, so using RLIKE pattern will be flexible enough to tackle mix of group table name values (i.e. w/o locale part the case on toggling multilingual mode). Will create a separate PR with this fix also with UT to use ```getRecipients()``` on multilingual test env.

Reply:
@monishdeb I wonder what we could do to see what happens when we run our WHOLE SUITE in another lingua

Reply:
@eileenmcnaughton I have submitted a PR https://github.com/civicrm/civicrm-core/pull/11924 in this regard. Extended a UT to ensure that switching to multilingual mode in and then reusing the existing mail draft won't affect recipient list building.